Week-7/StackAndQueue/7.5.js

```diff
@@ -0,0 +1,38 @@
+// Next Greater ELement for each array element
+function nextGreaterElement(arr, n) {
+  let stack = [];
+  let answer = [];
+  stack.push(arr[0]);
+
+  for (let i = 1; i < n; i++) {
+    if (stack.length == 0) {
+      stack.push(arr[i]);
+      continue;
+    }
+
+    /* 
+         Checking if current element is greater than stack top --
+         if yes, we push the element into answer array and pop
+         element from stack. We keep doing it until current element 
+         is lesser than any value in stack.  
+      */
+    while (stack.length !== 0 && stack[stack.length - 1] < arr[i]) {
```

Comment:
are you sure, we are having time complexity of O(N)?

Reply:
@shiv123-max Without resolving all of the comments, why did you merge your PR, I will not approve your PR from next time, I will request changes on your PR if you are going to merge PR without resolving all of the comments on PR.